templates/interface/Debian.erb

```diff
@@ -99,6 +102,31 @@ allow-hotplug <%= @interface %>
 <% if @vlan_raw_device -%>
     vlan-raw-device <%= @vlan_raw_device %>
 <% end -%>
+<% if @additional_networks -%>
+<% if @additional_networks.is_a? Array -%>
```

Comment:
No, that's ruby for you -- it tries to follow English syntax. [#is_a?](https://ruby-doc.org/core-2.4.1/Object.html#method-i-is_a-3F) method checks if object is of given class, not that object is an array. Kinda misleading, I also stumbled on this before.